src/block/tabs/edit.js

```diff
@@ -0,0 +1,188 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames'
+import {
+	InspectorStyleControls, InspectorTabs, PanelAdvancedSettings, AdvancedRangeControl,
+} from '~stackable/components'
+import {
+	BlockDiv,
+	useGeneratedCss,
+	MarginBottom,
+	Alignment,
+	Advanced,
+	CustomCSS,
+	Responsive,
+	CustomAttributes,
+	EffectsAnimations,
+	ConditionalDisplay,
+	Separator,
+	Transform,
+	ContentAlign,
+} from '~stackable/block-components'
+import {
+	withBlockAttributeContext,
+	withBlockWrapperIsHovered,
+	withQueryLoopContext,
+} from '~stackable/higher-order'
+import { useBlockContext } from '~stackable/hooks'
+import { version as VERSION, i18n } from 'stackable'
+
+/**
+ * WordPress dependencies
+ */
+import { compose } from '@wordpress/compose'
+import { __ } from '@wordpress/i18n'
+import { InnerBlocks } from '@wordpress/block-editor'
+import { TabsStyle } from './style'
+import { useEffect } from '@wordpress/element'
+import { dispatch } from '@wordpress/data'
+import { getBlockFromExample } from '@wordpress/blocks'
+
+const TEMPLATE = [
+	[ 'stackable/tab-labels', {} ],
+	[ 'stackable/tab-content', {} ],
+]
+
+const Edit = props => {
+	const {
+		className,
+		isSelected,
+		attributes,
+	} = props
+
+	// const [ tabsOptions, setTabsOptions ] = useState( [] )
+
+	useGeneratedCss( props.attributes )
+
+	const {
+		hasInnerBlocks, innerBlocks,
+	} = useBlockContext()
+	const {
+		insertBlock, removeBlocks, updateBlockAttributes,
+	} = dispatch( 'core/block-editor' )
+
+	const blockClassNames = classnames( [
+		className,
+		'stk-block-tabs',
+	] )
+
+	// useEffect( () => {
+	// 	for ( let i = 0; i < attributes.tabCount; i++ ) {
+	// 		setTabsOptions( [
+	// 			...tabsContent,
+	// 			{ value: i + 1, label: 'Tab' + ( i + 1 ) },
+	// 		]
+	// 		)
+	// 	}
+	// }, [ ] )
+
+	useEffect( () => {
+		if ( props.attributes.tabCount ) {
+			const tabContentBlock = document.querySelector( `[data-block="${ innerBlocks[ 1 ].clientId }"]` )
+			const columns = tabContentBlock.querySelectorAll( '[data-type="stackable/column"]' )
+			columns.forEach( ( element, index ) => {
+				if ( index !== 0 ) {
+					element.style.display = 'none'
+				}
+			} )
+		}
+	}, [ props.attributes.tabCount ] )
```

Comment:
Please see my previous comment about not directly manipulating CSS aside if it's the last resort.